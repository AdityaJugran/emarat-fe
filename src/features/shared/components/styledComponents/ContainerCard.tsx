import tw from 'twin.macro';
import styled from 'styled-components';

export default styled.div`
	${tw`
		mx-auto
		bg-white
		shadow-lg
		p-2
		h-full
        sm:mt-2
	`}
	width: ${(props: { width?: string }) => props.width || ''};
`;
