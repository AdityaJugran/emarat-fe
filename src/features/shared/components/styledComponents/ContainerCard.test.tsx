import renderer from 'react-test-renderer';
import ContainerCard from 'features/shared/components/styledComponents/ContainerCard';

it('Styled Title Snapshot', () => {
	const tree = renderer.create(<ContainerCard />).toJSON();
	expect(tree).toMatchSnapshot();
});
