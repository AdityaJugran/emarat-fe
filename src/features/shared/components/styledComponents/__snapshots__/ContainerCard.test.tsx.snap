// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Title Snapshot 1`] = `
<div
  className="sc-bdfBQB hqlFOK"
/>
`;
